src/commands/Staff/punishment.ts

```diff
@@ -134,6 +134,7 @@ const command: Command = {
 			if (!checkPermissions(interaction.member as Discord.GuildMember, punishment)) throw "You don't have permission to give this punishment"
 			if (user.bot) throw "You cannot punish bots!"
 			if (member?.roles.cache.has("768435276191891456")) throw "You cannot punish this user!" //Discord Staff
+			if (points! < (punishment.activePunishmentPoints ?? 0)) throw "You cannot punish this user with less points than they have!"
```

Comment:
We shouldn’t allow equal points either. Also feel free to tweak this message a little if you have better ideas but it should include the amount of points. 
```suggestion
			if (points! <= (punishment.activePunishmentPoints ?? 0)) throw `This user currently has an active punishment with ${punishment.activePunishmentPoints} points, so you cannot give them a less severe punishment!`
```

Reply:
The message is ok.